mock/src/components/Data/MockData.tsx

```diff
@@ -0,0 +1,1080 @@
+    "2020",
+    "217156",
+    "Brown University",
+    "58",
+    "brown-university",
+    "0.018764154",
+    "Men",
+    "1",
+  ],
+  [
+    "American Indian or Alaska Native",
+    "2020",
+    "2020",
+    "217156",
+    "Brown University",
+    "4",
+    "brown-university",
+    "0.00129408",
+    "Men",
+    "1",
+  ],
+  [
+    "Non-resident Alien",
+    "2020",
+    "2020",
+    "217156",
+    "Brown University",
+    "327",
+    "brown-university",
+    "0.105791006",
+    "Men",
+    "1",
+  ],
+  [
+    "White",
+    "2020",
+    "2020",
+    "217156",
+    "Brown University",
+    "691",
+    "brown-university",
+    "0.223552248",
+    "Men",
+    "1",
+  ],
+  [
+    "Asian",
+    "2020",
+    "2020",
+    "217156",
+    "Brown University",
+    "235",
+    "brown-university",
+    "0.076027176",
+    "Women",
+    "2",
+  ],
+  [
+    "Black or African American",
+    "2020",
+    "2020",
+    "217156",
+    "Brown University",
+    "95",
+    "brown-university",
+    "0.03073439",
+    "Women",
+    "2",
+  ],
+  [
+    "Native Hawaiian or Other Pacific Islanders",
+    "2020",
+    "2020",
+    "217156",
+    "Brown University",
+    "4",
+    "brown-university",
+    "0.00129408",
+    "Women",
+    "2",
+  ],
+  [
+    "Hispanic or Latino",
+    "2020",
+    "2020",
+    "217156",
+    "Brown University",
+    "207",
+    "brown-university",
+    "0.066968619",
+    "Women",
+    "2",
+  ],
+  [
+    "Two or More Races",
+    "2020",
+    "2020",
+    "217156",
+    "Brown University",
+    "85",
+    "brown-university",
+    "0.027499191",
+    "Women",
+    "2",
+  ],
+  [
+    "American Indian or Alaska Native",
+    "2020",
+    "2020",
+    "217156",
+    "Brown University",
+    "7",
+    "brown-university",
+    "0.002264639",
+    "Women",
+    "2",
+  ],
+  [
+    "Non-resident Alien",
+    "2020",
+    "2020",
+    "217156",
+    "Brown University",
+    "281",
+    "brown-university",
+    "0.090909091",
+    "Women",
+    "2",
+  ],
+  [
+    "White",
+    "2020",
+    "2020",
+    "217156",
+    "Brown University",
+    "660",
+    "brown-university",
+    "0.213523132",
+    "Women",
+    "2",
+  ],
+];
+
+/** A String[][] containing a small set of CSV data. */
+const mockSmall = [
+  ["Name", "Age", "Country"],
+  ["John", "30", "USA"],
+  ["Emily", "25", "Canada"],
+  ["Michael", "35", "UK"],
+  ["Sophia", "28", "France"],
+];
+
+/** A String[][] containing a small set of CSV data. */
+const mockSmall2 = [
+  ["Apple", "1.25", "10"],
+  ["Banana", "0.75", "15"],
+  ["Orange", "1.0", "12"],
+  ["Grapes", "2.5", "8"],
+];
+
+/** A String[][] containing a set of CSV data with on column. */
+const oneColumn = [["Only Column"], ["bingo"], ["bongo"], ["mingo"], ["mongo"]];
+
+/** A String[][] containing a set of empty CSV data. */
+const empty: any[] = [];
+
+/** A Map<String, Array<Array<String>>> mapping filename to the file data above. */
+export const mockData = new Map<String, Array<Array<String>>>([
```

Comment:
Nitpick: consider putting this at the top of the file so that the reader knows what is actual contained in the file so they don't have to scroll so far, especially because this is the thing that is most used and most likely to be editted in this file. Also could consider putting each (large) csv const Array<Array<STRING>>> into its own file and importing it